server/index.js

```diff
@@ -1,18 +1,38 @@
 const express = require('express');
 const cors = require('cors')
+const petModel = require('./db/index');
+const mongoose = require('mongoose');
 
 const app = express();
 const port = 8000;
 
 app.use(cors())
 app.use(express.json())
 
+mongoose.connect('mongodb://localhost:27017/pet-cards', () => console.log('Connected to DB'))
```

Comment:
Also, the URL of MongoDB should come from environment varialbes . Please read about it.

Reply:
Done it with dotenv package